test/graphio.jl

```diff
@@ -145,6 +145,12 @@ end
     @test length(loadgraphs(fname, NETFormat())) == 1
 end
 
+@testset "LGCompressed" begin
+    using CodecZlib
+    for g in values(allgraphs)
+        readback_test(LGCompressedFormat(), g; testfail=true)
```

Comment:
Shouldn't we also have some testst with `testfail=false` here?

Reply:
no. `testfail=true` adds tests to the default set. `testfail=false` (the default) skips the failure tests.